hack/build-image

```diff
@@ -196,7 +198,13 @@ def container_build(cli, target):
     if "docker" in args[0]:
         if target.arch != host_arch():
             raise RuntimeError("Docker does not support --arch")
-    else:
+    elif target.arch != host_arch() or FORCE_ARCH_FLAG:
+        # We've noticed a few small quirks when using podman with the --arch
+        # option. The main issue is that building the client image works
```

Comment:
Were you able to find any links to existing issues that can be recorded here?

Reply:
No, I just kept hitting this while working on the releases. I probably should look up if there's a podman issue for this and or report it. Regardless, the issue exists.